src/bindings/python/build_linux.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
this scripts is run within the `manylinux` container to build `libvalhalla` against an old version of `glib`